### PR TITLE
Document getting audio devices

### PR DESCRIPTION
**Pull request by SuaYoo**
Documents getting audio input and output devices.

## 📝 To Do

- [x] All linters pass
- [x] All tests pass
- [ ] Change documentation based on my changes

## ✋ Manual testing

1. Run `npm run build` in `packages/js`
2. Run `npm start` in `packages/js/examples/react-audio`
3. Navigate to http://localhost:6006/?path=/story/utilities--example
4. Click "Get Audio Input Devices". Verify your input devices are displayed
5. Click "Get Audio Output Devices". Verify your input devices are displayed (see note about browser compatibility below)

### Desktop

- [ ] Edge (latest)
- [x] Chrome
- [x] Firefox
- [ ] Safari

Firefox doesn't return an audio output device list. I've documented this in the comments.

## 📸 Screenshots

Firefox:
Screen Shot 2020-11-18 at 2 19 45 PM
Screen Shot 2020-11-18 at 2 19 49 PM

Chrome:
Screen Shot 2020-11-18 at 2 19 59 PM
Screen Shot 2020-11-18 at 2 20 06 PM

### CODE REVIEW

File: packages/js/examples/react-audio/stories/components/Utilities.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+  const initClient = () => {
+    clientRef.current = new TelnyxRTC({
+      env: environment,
+      login: username,
+      password,
+    });
+  };
+
+  const connect = async () => {
```
Comment (SuaYoo) [inline packages/js/examples/react-audio/stories/components/Utilities.jsx:17]: I initially thought I had to connect to use `getAudioInDevices` but that wasn't the case. I just left it in this PR for future use.

File: packages/js/examples/react-audio/stories/components/Utilities.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        setMessage('Connected');
+      });
+
+      clientRef.current.on('telnyx.error', () => {
+        setMessage('Received error attempting to connect');
+      });
+
+      await clientRef.current.connect();
+
+      clientRef.current = clientRef.current;
```
Comment (DeividVeloso) [inline packages/js/examples/react-audio/stories/components/Utilities.jsx:39]: Why do you need this?

File: packages/js/examples/react-audio/stories/components/Utilities.jsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+    setMessage(
+      `Audio output devices: ${
+        results.length ? results.map(({ label }) => label).join(', ') : 'None'
+      }`
+    );
+  };
+
+  if (!clientRef.current && environment && username && password) {
+    initClient();
```
Comment (DeividVeloso) [inline packages/js/examples/react-audio/stories/components/Utilities.jsx:66]: Why do you need this? I saw that you already call this function inside `connect()` line 26.

**Review by DeividVeloso — COMMENTED** [review]
It looks good, I just put some comments.

File: packages/js/src/Modules/Verto/webrtc/helpers.ts [hunk 4]
```diff
@@ -33,9 +33,13 @@ const _constraintsByKind = (
  * Retrieve device list using the browser APIs
  * If 'deviceId' or 'label' are missing it means we are on Safari (macOS or iOS)
  * so we must request permissions to the user and then refresh the device list.
+ *
+ * @NOTE Firefox has yet to fully implement audio output devices. As of v63,
```
Comment (hugobdev) [inline packages/js/src/Modules/Verto/webrtc/helpers.ts:37]: I think you can just write `Browser Compatibility Note:` just like you did in `packages/js/src/Modules/Verto/BrowserSession.ts`

**Review by hugobdev — APPROVED** [review]

File: packages/js/examples/react-audio/stories/components/Utilities.jsx (hunk 3 above)
Comment (SuaYoo) [reply packages/js/examples/react-audio/stories/components/Utilities.jsx:64]: `connect` is called when the button is clicked to re-initialize the client

**Pull request merged**